README.md

```diff
@@ -11,50 +15,31 @@ If you want to suggest a new feature for the C# or Visual Basic languages go her
 
-## Contribute!
+### Contributing
 
-Some of the best ways to contribute are to try things out, file bugs, and join in design conversations. 
+All work on the C# and Visual Basic compiler happens directly on [GitHub](https://github.com/dotnet/roslyn). Both core team members and external contributors send pull requests which go through the same review process.
 
-### Questions
+If you are interested in fixing issues and contributing directly to the code base, a great way to get started is to ask some questions on [GitHub Discussions](https://github.com/dotnet/roslyn/discussions)! Then check out our [contributing guide](https://github.com/dotnet/roslyn/blob/master/docs/wiki/Contributing-Code.md) which covers the following:
 
-A great way to get started is to ask some questions!
-- Start with a question on [discussions](https://github.com/dotnet/roslyn/discussions)
-- You can also join in on the design discussions on [gitter](https://gitter.im/dotnet/roslyn) or [discord](http://aka.ms/discord-csharp-roslyn)
+- [Coding guidelines](https://github.com/dotnet/roslyn/blob/master/docs/wiki/Contributing-Code.md)
+- [The development workflow, including debugging and running tests](https://github.com/dotnet/roslyn/blob/master/docs/contributing/Building%2C%20Debugging%2C%20and%20Testing%20on%20Windows.md)
+- [Submitting pull requests](https://github.com/dotnet/roslyn/blob/master/CONTRIBUTING.md)
+- Finding a bug to fix in the [IDE](https://aka.ms/roslyn-ide-bugs-help-wanted) or [Compiler](https://aka.ms/roslyn-compiler-bugs-help-wanted)
+- Finding a feature to implement in the [IDE](https://aka.ms/roslyn-ide-feature-help-wanted) or [Compiler](https://aka.ms/roslyn-compiler-feature-help-wanted)
 
-### See if your issue is already being worked on! (Add your own votes using the 👍 reaction)
-- [IDE](https://aka.ms/roslyn-ide-in-progress)
-- [Compiler](https://aka.ms/roslyn-compiler-in-progress)
+### Community
 
-### Vote in the Backlog! (Add your own votes using the 👍 reaction)
-- [IDE Bugs](https://aka.ms/roslyn-ide-bug-backlog)
-- [IDE Features](https://aka.ms/roslyn-ide-feature-backlog)
-- [Compiler Bugs](https://aka.ms/roslyn-compiler-bug-backlog)
-- [Compiler Features](https://aka.ms/roslyn-compiler-features-backlog)
+The Roslyn community can be found on [GitHub Discussions](https://github.com/dotnet/roslyn/discussions), where you can ask questions, voice ideas, and share your projects.
 
-### Find a bug to fix! (Add your own votes using the 👍 reaction)
-- First read this guide: [How to Contribute](docs/wiki/Contributing-Code.md)
-- [Building, testing and debugging the sources](docs/wiki/Building-Testing-and-Debugging.md)
-- Top Bugs 
-  - [IDE](https://aka.ms/roslyn-ide-bugs-help-wanted)
-  - [Compiler](https://aka.ms/roslyn-compiler-bugs-help-wanted)
+To chat with other community members, you can join the Roslyn [Gitter](https://gitter.im/dotnet/roslyn) or [Discord](https://discord.com/invite/tGJvv88).
```

Comment:
I'd put Discord ahead of Gitter. That is the more actively engaged community at this point.